### PR TITLE
Closes #155: Fix components reaching past screen size then requiring scrolling

### LINKED ISSUE #155: Components are larger than the screen width
**Describe the bug**
Some views are larger than the size of the screen, so the user has to scroll down to see the rest

**To Reproduce**
1. Go to the main app page (path '/')
2. Go to the full screen calendar widget (path '/calendar')

**Expected behavior**
The component should not go beyond the screen size. It should be dynamically sized to fit below the top bar.

**Screenshots**

image

image

**Desktop (please complete the following information):**
 - OS: [mac OS]
 - Browser [Firefox]

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by RawiriHohepa**
Closes #155 

## Proposed Changes

- Add flex styling to App to dynamically size the widgets/journal div
- Remove view height based styling of calendar widget in favour of styling based on parent's height

### CODE REVIEW

File: telos-frontend/src/components/calendar/DashboardCalendar.jsx [hunk 1]
```diff
@@ -50,14 +50,14 @@ const DashboardCalendar = () => {
   ]);
 
   return (
-    <div>
+    <>
```
Comment (lhuijser) [inline telos-frontend/src/components/calendar/DashboardCalendar.jsx:53]: Not an issue, just curious, but what does using this instead of div do?

**Review by lhuijser — APPROVED** [review]
Looks great to me! Just one question but that doesn't impact the functionality, so more than happy to approve :)

File: telos-frontend/src/components/calendar/DashboardCalendar.jsx (hunk 1 above)
Comment (milkyway404) [reply telos-frontend/src/components/calendar/DashboardCalendar.jsx:53]: After googling, it seems like a type that's not defined, basically used to wrap everything. Not sure why a div isn't used though.

**Review by milkyway404 — COMMENTED** [review]
Looks pretty good, just not sure if a div would work better?

**Review by JChamberlain28 — APPROVED** [review]
LGTM 🤠

File: telos-frontend/src/components/calendar/DashboardCalendar.jsx (hunk 1 above)
Comment (RawiriHohepa) [reply telos-frontend/src/components/calendar/DashboardCalendar.jsx:53]: The div was only as big as it needed to be to fit its contents - the two options were to either use an empty fragment or to style the div to have height of 100%, either one works

**Pull request merged**